### ISSUE TITLE
Bad solutions file with no solutions and certain flags

### ISSUE THREAD

**Issue by JBeaddie** [conjure-cp/conjure#541]
Assume that impossible.essence is an essence file with no solutions.

Then running:
```
conjure solve impossible.essence --solutions-in-one-file --output-format=json
```

Conjure will still create a _model000001.solutions.json_ file.
In addition the solutions file will only contain:
```
]
```
A right square-bracket.

**Comment by ozgurakgun** [comment]
I think it probably should still create the file but have [] in it, what do you think?

**Comment by ozgurakgun** [comment]
I've implemented this in #542
